components/frontend/src/utils.js

```diff
@@ -322,6 +322,18 @@ export function scaled_number(number) {
     return (number / Math.pow(1000, exponent)).toFixed(0) + scale[exponent]
 }
 
+export function formatMetricValue(scale, value) {
+    // metric value may be a string (version number or percentage), float, or integer
```

Comment:
The comment is a bit misleading, I guess it applies to the check on line 330?

Reply:
Replaced the comment by propTypes.